airflow/providers/slack/CHANGELOG.rst

```diff
@@ -24,6 +24,19 @@
 Changelog
 ---------
 
+6.1.0
```

Comment:
Should be  with 7.0.0 ( there is an entry in change log)

Reply:
Oh thanks. I was going to review those today as well :)